docs/ci: improvements (#38)
bank-vaults/vault-secrets-reloader: deploy/charts/vault-secrets-reloader/README.md

| `service.externalPort` | int | `443` | Reloader service external port |
| `service.internalPort` | int | `8443` | Reloader service internal port |
| `service.name` | string | `"vault-secrets-reloader"` | Reloader service name |
| `service.type` | string | `"ClusterIP"` | Reloader service type |
| `serviceAccount.annotations` | object | `{}` | Annotations to add to the service account |
| `serviceAccount.create` | bool | `true` | Specifies whether a service account should be created |
| `serviceAccount.name` | string | `""` | The name of the service account to use. If not set and create is true, a name is generated using the fullname template |
| `tolerations` | list | `[]` | List of node tolerations for the pods. Check: https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/ |
| `volumeMounts` | list | `[]` | Extra volume mounts for Reloader deployment |
| `volumes` | list | `[]` | Extra volume definitions for Reloader deployment |

Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.

### Time intervals

> Configure the time intervals between two runs of the `collector` and `reloader` workers with a value in Go Duration format:

Reviewer: ```suggestion
Configure the time intervals for periodic `collector` and `reloader` workers with a value in Go Duration format:
```
